### ISSUE TITLE
Redoc-cli options pass as json file.

### ISSUE THREAD

**Issue by BusinessDuck** [Redocly/redoc#754]
How to pass options to redoc-cli as separated json file? 

```
redoc-cli ./scheme.json --options ./redoc.json
```
redoc.json listing
```
{
    nativeScrollbars: true,
    theme: { colors: { main: "#dd5522" } },
}

**Comment by knidarkness** [comment]
Here is PR for this issue: #1049

**Comment by RomanHotsiy** [comment]
Closed in #1049

**Comment by richartkeil** [comment]
Mh, this is not yet published with npm is it?
